src/coalesce-vue-vuetify3/src/components/input/c-datetime-picker.vue

```diff
@@ -627,6 +627,39 @@ function emitInput(value: Date | null) {
   }
 }
 
+function isDateAllowed(date: Date) {
```

Comment:
Maybe this should just be axed and if you click 'today' when there is a min/max value and today doesn't fall within it we just clamp to be within the bounds?